src/Symfony/Component/Notifier/Bridge/Matrix/README.md

```diff
@@ -0,0 +1,33 @@
+Matrix Synapse Notifier
+=======================
+
+Provides [Matrix Synapse](https://matrix.org) integration for Symfony Notifier.
+It uses the [Matrix Client-Server API](https://spec.matrix.org/v1.13/client-server-api/).
+
+DSN example
+-----------
+
+```
+MATRIX_DSN=matrix://HOST:PORT/?accessToken=ACCESSTOKEN&ssl=true
+```
+
+where:
+- `HOST` is your Matrix Synapse Hostname / IP ( without `http(s)://` )
+- `PORT` is your Matrix Synapse Port
+- `ACCESSTOKEN` is your Matrix Synapse Token (botuser).
+
+The `ssl` parameter is optional and defaults to `true`.
+
+Getting the ACCESSTOKEN
```

Comment:
```suggestion
MATRIX_DSN=matrix://HOST:PORT/?accessToken=ACCESS_TOKEN&ssl=true
```

where:
- `HOST` is your Matrix Synapse Hostname / IP ( without `http(s)://` )
- `PORT` is your Matrix Synapse Port
- `ACCESS_TOKEN` is your Matrix Synapse Token (botuser).

The `ssl` parameter is optional and defaults to `true`.

Getting the ACCESS_TOKEN
```